tests/test_interpolation.py

```diff
@@ -169,6 +169,42 @@ def test_precomputed_interpolation_time():
         assert np.allclose(sf.data[it, :], it)
 
 
+def test_precomputed_injection():
+    """Test injection with PrecomputedSparseFunction which accepts
+       precomputed values for interpolation coefficients
+    """
+    shape = (11, 11)
+    coords = [(.05, .95), (.45, .45)]
+    origin = (0, 0)
+    result = 0.25
+
+    r = 2  # Constant for linear interpolation
```

Comment:
nitpicking:
Comment should go before variable?

Reply:
I have now moved the comments before the variable